src/components/ConnectModal.tsx

```diff
@@ -8,18 +8,23 @@ interface ConnectModalProps {
   // represents the other user 'I' am trying to connect to.
   userToConnectTo: PublicUser;
 
-  handleEmailConect: () => void;
+  handleEmailConnect: (message: string) => void;
 
   closeModal: () => void;
 }
 
 const ConnectModal = (props: ConnectModalProps): JSX.Element => {
   const [isOpen, setIsOpen] = useState(true);
+  const [customMessage, setCustomMessage] = useState("");
 
   const onClose = () => {
     setIsOpen(false);
     props.closeModal();
   };
+  function handleMessageChange(event: ChangeEvent<HTMLTextAreaElement>): void {
```

Comment:
Super small point, but for consistency matters, let's try to use 

`const handleMessageChange = (event) => {...}`

Reply:
ok